js-foundations.js

```diff
@@ -3,28 +3,49 @@
 
 // 1. Write a function that takes a number as an argument and decides if the number is evenly divisble by three or not. Use the test cases 15, 0, 385, -7. Expected output: "15 is divisible by three", "0 is divisible by three", "385 is not divisble by three", "-7 is not divisble by three".
 
-
+var numbers = (num) =>{
+  if (num % 3 === 0){
+    return `${num} is divisble by 3.`
+  } else {
+    return `${num} is NOT divisble by 3.`
+  }
+}
+console.log(numbers(15));
+console.log(numbers(0));
+console.log(numbers(385));
+console.log(numbers(-7));
 
 // 2. Create an object called helloMe. Include your firstName, lastName and TWO OTHER properties of you. Add a method to your object that will return a sentence using the object properties to describe you.
 
-
+const helloMe = (firstName, lastName, height, eyeColor) => {
+  return `My name is ${firstName} ${lastName}, I have ${eyeColor} eyes and am ${height}.`
+}
+console.log(helloMe("Ofrey", "Barsheshet", "5'10", "brown"));
 
```

Comment:
This totally works, but the goal here is to work with objects and key:value pairs rather than just a function.